### PR TITLE
Rate limiter whitelist of IP address

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
As suggested in #792

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -225,4 +225,10 @@
 
     'iip_private' => env('IIP_PRIVATE', 'http://127.0.0.1:8002'),
     'iip_public' => env('IIP_PUBLIC', 'https://img.webumenia.sk'),
+
+    'rate_limiter' => [
+        'whitelisted_ip_addresses' => env('RATE_LIMITER_WHITELISTED_IP_ADDRESSES')
+            ? explode(',', env('RATE_LIMITER_WHITELISTED_IP_ADDRESSES'))
+            : [],
```
Comment (eronisko) [inline config/app.php:232]: Is there an actual need for the configurability? Could we just list the IPs here?

Comment (rastislav-chynoransky) [reply config/app.php:232]: It would be handy to be able to add our own IP addresses.

Comment (eronisko) [reply config/app.php:232]: Do you mean for development? Maybe we could disable it based on app environment then?

**Review by eronisko — APPROVED** [review]
I don't have any issues with the implementation, but I'm a little concerned that the whitelisting is potentially needed for development. In my opinion, the rate limiter should be a safety net, not a mode of operation.

My guess is that either
- the rate limiter is too strict, or
- the client implementation is not optimal, or
- the API needs a design adjustment

In any case, from an architecture perspective, I don't think the assumption that "nobody should call this so often, except for us" is 100% sound. The "whitelisted" calls are ultimately being triggered by visitors, so still outside of our control. In fact, for those apps -- that might potentially generate lots of requests -- we're losing that safety mechanism altogether.

My suggestion would be to increase the rate limit for everyone to a level we're comfortable with.

cc @igor-kamil @FrantisekMichalSebestyen

**Comment by rastislav-chynoransky** [comment]
When using SSR frontends the requests will come from our own server, that's why we need to make a whitelist here. And if high rates are an issue to implement rate limiter on the caller side.

**Review by igor-kamil — COMMENTED** [review]
for now just turn it off. in case of troubles, we can back to it

**Pull request closed without merging**